.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "vendor/mimalloc"]
 	path = vendor/mimalloc
 	url = https://github.com/microsoft/mimalloc
-	branch = v1.8.2
+	branch = master
```

Comment:
change to master branch?

Reply:
Yes, the master branch is the stable branch, [master: latest stable release (based on dev-slice).](https://github.com/microsoft/mimalloc/tree/db3d8485d2f45a6f179d784f602f9eff4f60795c?tab=readme-ov-file#branches)

1. Execute  `git submodule update --remote --recursive` and push the `vendor/mimalloc` folder changes when a submodule needs to be updated.
2. Executing `git submodule update --init --recursive` will not update the master branch's latest commit, It will pull the commit from step 1.

Reply:
in addition, `.gitmodules` does not support tag or commit